lte/gateway/python/magma/enodebd/device_config/configuration_init.py

```diff
@@ -48,6 +48,19 @@
         'bandwidth_mhz', 'cell_id',
         'allow_enodeb_transmit',
         'mme_address', 'mme_port',
+        'a1_threshold_rsrp', 'lte_a1_threshold_rsrq',
+        'hysteresis', 'time_to_trigger',
+        'a2_threshold_rsrp', 'lte_a2_threshold_rsrq',
+        'lte_a2_threshold_rsrp_irat_volte', 'lte_a2_threshold_rsrq_irat_volte',
+        'a3_offset', 'a3_offset_anr',
+        'a4_threshold_rsrp', 'lte_intra_a5_threshold_1_rsrp',
+        'lte_intra_a5_threshold_2_rsrp', 'lte_inter_anr_a5_threshold_1_rsrp',
+        'lte_inter_anr_a5_threshold_2_rsrp', 'b2_threshold1_rsrp',
+        'b2_threshold2_rsrp', 'b2_geran_irat_threshold',
+        'qrxlevmin_sib1', 'qrxlevminoffset',
+        's_intrasearch', 's_nonintrasearch',
+        'qrxlevmin_sib3', 'reselection_priority',
+        'threshservinglow', 'ciphering_algorithm', 'integrity_algorithm',
```

Comment:
If we are going to be adding this large number of params. We might want to reconsider using a namedtuple and use a dictionary with some hierarchy

Reply:
![image](https://user-images.githubusercontent.com/87301024/147403805-fe76f311-692b-4de7-9502-144150bde592.png)
Parameters have been integrated, thank you!